_posts/2022-05-01-release-table-store-0.1.0.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title:  "Apache Flink Table Store 0.1.0 Release Announcement"
+subtitle: "Unified streaming and batch store for building dynamic tables on Apache Flink."
+date: 2022-05-01T08:00:00.000Z
+categories: news
+authors:
+- Jingsong Lee:
+  name: "Jingsong Lee"
+
+---
+
+The Apache Flink Community is pleased to announce the preview release of the
+[Apache Flink Table Store](https://github.com/apache/flink-table-store) (0.1.0).
+
+Flink Table Store is a unified streaming and batch store for building dynamic tables
+on Apache Flink. It uses a full Log-Structured Merge-Tree (LSM) structure for high speed
+and large amount of data update & query capability.
+
+Please check out the full [documentation]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.1/) for detailed information and user guides.
+
+Note: Flink Table Store is still in beta status and undergoing rapid development,
+we do not recommend that you use it directly in a production environment.
+
+## What is Flink Table Store
+
+Open [Flink official website](https://flink.apache.org/), you can see the following line:
+`Apache Flink - Stateful Computations over Data Streams.` Flink focuses on distributed computing,
+which brings real-time big data computing. But pure computation doesn't bring value, users need
+to combine Flink with some kind of external storage.
+
+For a long time, we found that no external storage can fit Flink's computation model perfectly,
+which brings troubles to users. So Flink Table Store was born, it is a storage built specifically
+for Flink, for big data real-time update scenario. From now on, Flink is no longer just a computing
+engine.
+
+Flink Table Store is a unified streaming and batch table format:
+- As the storage of Flink, it first provides the capability of Queue.
+- On top of the Queue capability, it precipitates historical data to data lakes.
+- The data on data lakes can be updated and analyzed in near real-time.
+
+## Core Features
+
+Flink Table Store supports the following usage:
+- **Streaming Insert**: Write changelog streams, including CDC from database and streams.
+- **Batch Insert**: Write batch data as offline warehouse, including OVERWRITE support.
+- **Batch/OLAP Query**: Read snapshot of the storage, efficient querying of real-time data.
+- **Streaming Query**: Read changes of the storage, ensure exactly-once consistency.
```

Comment:
Nit: "changes of the storage" may be wordy and can be simplified as "storage changes".